runtime.md

```diff
@@ -103,6 +103,7 @@ This operation MUST generate an error if it is not provided the container ID.
 Attempting to start a container that does not exist MUST generate an error.
 Attempting to start an already started container MUST have no effect on the container and MUST generate an error.
 This operation MUST run the user-specified program as specified by [`process`](config.md#process).
+This operation MUST generate an error if `process.args` was not set.
```

Comment:
After some [discussion on IRC][1], @mikebrow recommended the following addition to the `process.args` docs:

> When start is used process.args is no longer OPTIONAL and MUST be set.

There's also a similar conditional requirement [in the device spec][2], but “REQUIRED unless you won't call `start`” and “OPTIONAL unless you'll call `start` in which case it's required” both feel too awkward.

With 7071706 → e45ddba I've added:

> When [`start`](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/runtime.md#errors) will be called, this property is REQUIRED.

which gets the cross-reference to the `start` command.  It replaces @mikebrow's “is no longer OPTIONAL and MUST be set” with “is REQUIRED”, because I like “REQUIRED” better than “MUST be set”.  I think “is no longer OPTIONAL” is fairly clear from the context, but I'm ok adding that back in if folks feel like it's necessary.  And I've replaced “is used” with “will be called” because you need to [set this field at `create` time][3] in order to use it when `start` is called.

[1]: http://ircbot.wl.linuxfoundation.org/eavesdrop/%23opencontainers/%23opencontainers.2017-01-11.log.html#t2017-01-11T18:34:39
[2]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc3/config-linux.md#L118
[3]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc3/runtime.md#L97